src/Components/Web.JS/src/Platform/Mono/MonoDebugger.ts

```diff
@@ -42,6 +44,51 @@ export function attachDebuggerHotkey(resourceLoader: WebAssemblyResourceLoader):
   });
 }
 
+async function launchFirefoxDebugger() {
+  const appDiv = document.getElementById('app');
+    const response = await fetch(`_framework/debug?url=${encodeURIComponent(location.href)}&isFirefox=${isFirefox}`);
+    if (response.status !== 200) {
+      const warningToDebug = document.createElement('div');
+      warningToDebug.id = 'warningToDebug';
+      warningToDebug.style.padding = '20px';
+      warningToDebug.style.backgroundColor = '#f44336';
+      warningToDebug.style.color = 'white';
+      warningToDebug.style.opacity = '1';
+      warningToDebug.style.transition = 'opacity 0.6s';
+      warningToDebug.style.marginBottom = '15px';
+      warningToDebug.style.top = '0';
+      warningToDebug.style.right = '0';
+      warningToDebug.style.position = 'absolute';
+      warningToDebug.style.zIndex = '2';
+      warningToDebug.innerHTML = await response.text();
+      const closeWarningButton = document.createElement('span');
+      closeWarningButton.innerHTML = '&times;';
+      closeWarningButton.style.marginLeft = '15px';
+      closeWarningButton.style.color = 'white';
+      closeWarningButton.style.fontWeight = 'bold';
+      closeWarningButton.style.float = 'right';
+      closeWarningButton.style.fontSize = '22px';
+      closeWarningButton.style.lineHeight = '20px';
+      closeWarningButton.style.cursor = 'pointer';
+      closeWarningButton.style.transition = '0.3s';
+      closeWarningButton.onclick = function(){
+        warningToDebug.style.opacity = '0';
+        setTimeout(function() {
+          warningToDebug.style.display = 'none';
+        }, 600);
+      };
+      warningToDebug.appendChild(closeWarningButton);
```

Comment:
This is quite a lot of code to include if this is only for displaying an error about debugging. Since we have to include all this code even in production bundles, would it be viable to shrink it? For example we don't need an error to be nicely styled. I think it would be enough even just to use `alert(someMessage)`. Or if it's unlikely this ever occurs, we could just `console.error(someMessage)`.

Reply:
> This is quite a lot of code to include if this is only for displaying an error about debugging. Since we have to include all this code even in production bundles, would it be viable to shrink it? For example we don't need an error to be nicely styled. I think it would be enough even just to use `alert(someMessage)`. Or if it's unlikely this ever occurs, we could just `console.error(someMessage)`.

This will probably happen usually, but if you think a console.warning is better for sure I can change it :)

Reply:
I would ideally prefer to start with something minimal like output to console (or `alert` if it's desirable to be visible in the main UI).

Reply:
Done :)